### PR TITLE
Adds a new type of revolver, the DonkCo. New Syndie. 

### PR DESCRIPTION
**Pull request by Enbyy**
## About The Pull Request
(big thanks to Sans and Orfis for helping with the descriptions and names)

Hey there space cowboy, DonkCo. here. Tired of using the same old revolver to blast Nanotransen crewmembers? Well don't you worry, we got just the gun for you! 

Introducing the  DonkCo. New Syndie!

image
image
The DonkCo. New Syndie has a 7 shot internal magazine that utilizes .38 Special Hot Load rounds with a match trigger and extended hammer perfect for fanning. This bad girl can shoot rounds so fast, NandoTramson Security won't know what hit em. You need more rounds? Hack your autolathe or buy more from the uplink! DonkCo. got you covered!

image
image
 Now don't take my word for it. Here's some special DonkCo. test footage to prove it!

https://user-images.githubusercontent.com/76571156/233742070-48b6525b-9863-4a9c-84d9-bf730c861aad.mp4

Isn't that something? Buy your DonkCo. New Syndie Today!
## Why It's Good For The Game

 It's a slightly less effective side grade to the revolver. It takes on average 1-2 more hits to kill, but it shoots incredibly fast. If you can hit the 3-5 bullets you need, this thing will kill incredibly fast. But to do that, you need to be in the 1-3 tile range as it's very inaccurate and it's hard to do that against moving targets. Therefore, this gun trades the range and versatility the syndie revolver has for a slightly faster TTK, faster fire rate, and easier access to it's ammo. It's also just a incredibly fun gun to use. It's really fun to just pump out lead, even if half the shots are missing.
 
 Also Glup Shizzo rootin tootin cowboy arc lets go
## Changelog

:cl:
add: The DonkCo. New Syndie Revolver, using .38 Special Hot Load ammo
add: .38 Special Hot Load ammo and associated Speedloaders
/:cl:

### CODE REVIEW

**Review by flowercuco — CHANGES_REQUESTED** [review]
i think that the revolver and speedloader could just be in the same file, a maintainer can just split them if we need an "ammunition" file for traitors and a separate "wapoons" one

**Comment by flowercuco** [comment]
unrelated and not your fault, i fucking hate how guns work so bad

File: orbstation/code/objects/machinery/autolathe_overrides.dm [hunk 1]
```diff
@@ -15,3 +15,15 @@
 		RND_CATEGORY_WEAPONS + RND_SUBCATEGORY_WEAPONS_AMMO,
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SECURITY
+
+/datum/design/special_hotload
+	name = ".38 Special Hot Load bullet"
```
Comment (flowercuco) [inline orbstation/code/objects/machinery/autolathe_overrides.dm:20]: add (VERY LETHAL) to this to match the current changes even though security cant realistically use this item

**Review by flowercuco — CHANGES_REQUESTED** [review]

File: orbstation/code/objects/machinery/autolathe_overrides.dm (hunk 1 above)
Comment (Enbyy) [reply orbstation/code/objects/machinery/autolathe_overrides.dm:20]: add it where?

**Comment by Enbyy** [comment]
> unrelated and not your fault, i fucking hate how guns work so bad

yeah lol it kinda sucks

**Comment by Enbyy** [comment]
> i think that the revolver and speedloader could just be in the same file, a maintainer can just split them if we need an "ammunition" file for traitors and a separate "wapoons" one

yeah that's fair, i can do that

**Review by flowercuco — APPROVED** [review]

**Pull request merged**